styles-conference/assets/stylesheets/styles.css

```diff
@@ -395,3 +389,141 @@ textarea {
 #form_ul {
   padding-left: 24px;
 }
+
+@media screen and (min-device-width: 768px) {
+  .content {
+    width: 67%;
+  }
+
+  .app-wrap {
+    align-items: center;
+  }
+
+  #welcome {
+    background-image: url(/styles-conference/assets/desktop_bg.svg);
+    background-size: contain;
+    background-position: top;
+  }
+
+  header {
+    width: 100%;
+  }
+
+  .card {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    height: auto;
+  }
+
+  .card_image {
+    width: 100%;
+    height: auto;
+  }
+
+  #tonic1 {
+    order: 1;
+  }
+
+  #stories1 {
+    order: 2;
+  }
+
+  #img_link_storie1,
+  #img_link_storie2 {
+    order: 1;
+  }
+
+  .canopy,
+  .text_li {
+    font-size: 18px;
+  }
+
+  .card_bottom {
+    margin-top: 6%;
+  }
+
+  .card_bottom > * {
+    flex: none;
+  }
+
+  .categories {
+    margin-top: 5%;
+    margin-bottom: 5%;
+  }
+
+  #menu_list {
+    visibility: visible;
+    float: right;
+    display: flex;
+    margin-right: 130px;
+    gap: 15px;
+  }
+
+  #menu_list > * > * {
+    font-size: 15px;
+    font-weight: 500;
+  }
+
+  .menu {
+    visibility: hidden;
+  }
+
+  .nav-bar {
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  #work {
+    grid-template-rows: repeat(4, auto);
+  }
+
+  .about_my_self {
+    display: flex;
+  }
+
+  .content_about_self2 {
+    padding: 1px;
+    margin: 1px;
+  }

```

Comment:
- [ ] Kindly make sure the "Languages, Frameworks & Skills" section has a maximum of `60%` width to make it look more like the [Figma design](https://github.com/Mar12358/Portfolio/pull/10/files#diff-4d23d66c0aa65c0d192c66d068a6b247bb993e2e57ca83c5a98317c94f0838ebR394-R397)🎨

Reply:
- [ ] This issue is not yet resolved :x: 


- [ ] According to the previous reviewer's suggested change, kindly ensure the with is 60% to make your design look more like the Figma design :+1: